pom.xml

```diff
@@ -50,10 +50,10 @@
     </issueManagement>
 
     <properties>
-        <jenkins.version>1.580.1</jenkins.version>
-        <!--<jenkins.version>2.46.3</jenkins.version>-->
-        <java.level>6</java.level>
-        <!--<java.level>7</java.level>-->
+        <jenkins.version>2.60.2</jenkins.version>
```

Comment:
Please do not increase version, because that's remove support of old version of Jenkins.